pkg/doc/bbs/bbs12381g2pub/internal/kilic/bls12-381/g1.go

```diff
@@ -0,0 +1,176 @@
+/*
+Taken from https://github.com/kilic/bls12-381/blob/master/g1.go
+(rev a288617c07f1bd60613c43dbde211b4a911e4791)
+
+Changes (placed separately at g1_custom.go).
+1) Use black2b as hash function for HashToCurve().
+2) Custom implementation of osswuMap() (this algorithm is re-written from Rust code
```

Comment:
Does this mean that an additional change to kilic is needed? Is there a PR for that at kilic?

Reply:
Yes. https://github.com/kilic/bls12-381/pull/25

Reply:
@kdimak What about the "Custom implementation of osswuMap"?

Reply:
That’s related to the second change - the algorithm difference between kilic and libursa. I haven’t made a PR regarding this.

Reply:
https://github.com/hyperledger/aries-framework-go/pull/2397#discussion_r542496317